resources/dicts/thoughts/alive/kitten.json

```
@@ -252,7 +393,14 @@
             "Wasn't paying attention and tripped",
             "Ate a bug",
             "Has SO much energy",
-            "Is zooming around"
+            "Is zooming around",
+            "Pounced on r_c",
+            "Is practicing {PRONOUN/m_c/poss} pounce on r_c's tail",
+            "Is complaining of a tummy ache after sneaking into the medicine cats herbs and eating one",
```

```suggestion
            "Is complaining of a tummyache after sneaking into the medicine den's herb storage and eating one",
```
can't really specify "medicine cat's" or "medicine cats'" since afaik there's no way to tell how many medicine cats there are from here